tests/rptest/perf/small_batches_test.py

```diff
@@ -0,0 +1,41 @@
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.services.openmessaging_benchmark import OpenMessagingBenchmark
+from rptest.services.openmessaging_benchmark_configs import OMBSampleConfigurations
+from rptest.services.cluster import cluster
+
+
+class SmallBatchesTest(RedpandaTest):
```

Comment:
How do these tests work with different hardware sizes? E.g., they require 6 nodes, but is 6 nodes enough if the instance sizes are small? Probably we require some minimum size of instance, and then also if the instances are much larger the test passes "easily".

This is basically out of scope for this change but I'm curious how this works.

Reply:
These nightly regression tests are solely targeted at a 3x `i3en.6xlarge` cluster. So the short answer is they don't work for different hardware sizes.

Reply:
Unfortunately we don't have a way of specifying hardware requirements in DT, only instance count.

Reply:
Thanks for this clarification.